common/src/main/java/io/pravega/common/hash/RandomFactory.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.hash;
+
+import java.security.SecureRandom;
+import java.util.Random;
+import javax.annotation.concurrent.GuardedBy;
+import lombok.Synchronized;
+
+public class RandomFactory {
```

Comment:
Please add Javadoc to this class in the common library, as per our Contribution Guidelines.

Reply:
Done